violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/EditMenu.java

```diff
@@ -78,10 +78,9 @@ public void actionPerformed(ActionEvent event) {
                     IEditorPart activeEditorPart = getActiveEditorPart();
                     IEditorPartBehaviorManager behaviorManager = activeEditorPart.getBehaviorManager();
                     List<UndoRedoCompoundBehavior> found = behaviorManager.getBehaviors(UndoRedoCompoundBehavior.class);
-                    if (found.size() != 1) {
-                        return;
+                    if (found.size() >= 1) {
```

Comment:
The problem is, that the conventions are mixed in the project. I think, that it should be consistent with the convention for methods and class.